linkedin2username.py

```diff
@@ -576,20 +579,21 @@ def clean(raw_list):
             clean_list.append(name)
 
     return clean_list
+  
+# Function needs to expect the variable so add it
+# old: def write_files(company, domain, name_list)
```

Comment:
```suggestion
  
```

These comments are useful in context of the PR, but they are not useful in the code itself. Can you remove them?